stock_move_packaging_qty/tests/test_stock_move_packaging_qty.py

```diff
@@ -59,3 +66,20 @@ def test_product_uom_qty_change(self):
             move_f.product_packaging_id = self.packaging
             self.assertEqual(move_f.product_uom_qty, 5)
             self.assertEqual(move_f.product_packaging_qty, 1)
+        picking = picking_f.save()
+        self.assertEqual(picking.state, "draft")
+        picking.action_assign()
+        picking.action_set_quantities_to_reservation()
```

Comment:
No need to test if `action_set_quantities_to_reservation()` is done correctly. Uses another fields that are not established in picking/move and the default values maybe differ from you expect.

```suggestion
        picking.move_line_ids[0].qty_done = picking.move_line_ids[0].reserved_uom_qty
```